cypress/components/dialog-full-screen/dialog-full-screen.cy.tsx

```diff
@@ -293,13 +287,19 @@ context("Testing DialogFullScreen component", () => {
         .should("have.css", "padding-right", "0px")
         .should("have.css", "padding-bottom", "0px");
     });
+
+    // skip this test for now as FE-6053 has not been fixed yet
+    it.skip("should render DialogFullScreen component with DisableClose prop", () => {
+      CypressMountWithProviders(<DialogFullScreenComponent disableClose />);
+      closeIconButton().should("be.disabled").and("have.attr", "disabled");
```

Comment:
we suppose to add that if we press/trigger this button the `DialogFullScreen` should remain opened